### PR TITLE
Class Version 5

### PR DESCRIPTION
**Pull request by GIT-REPOSITORY-TEST**
The changes are completed.. Please review

### CODE REVIEW

File: zsales_class_git.clas.abap [hunk 1]
```diff
@@ -12,11 +12,14 @@ CLASS zsales_class_git DEFINITION
         matnr TYPE matnr,
         charg TYPE charg_d,
         werks TYPE werks_d,
-        matkl TYPE matkl,
+        lgort TYPE lgort_d,  "New Change V4 - Branch 5
```
Comment (Sakthy-g) [inline zsales_class_git.clas.abap]: Specify table name in structure

File: zsales_class_git.clas.abap [hunk 2]
```diff
@@ -69,8 +72,11 @@ CLASS ZSALES_CLASS_GIT IMPLEMENTATION.
              matnr,
              charg,
              werks,
-             matkl,   "New Change V1 - Branch 1
-             erdat    "New Change V2 - Branch 2
+             lgort,    "New Change V4 - Branch 5
+             vstel,    "New Change V4 - Branch 5
```
Comment (Sakthy-g) [inline zsales_class_git.clas.abap]: Remove Shipping point - not required

**Review by Sakthy-g — CHANGES_REQUESTED** [review]
Sent you review comments - Please fix

File: zsales_class_git.clas.abap (hunk 1 above)
Comment (GIT-REPOSITORY-TEST) [reply zsales_class_git.clas.abap]: The above review comment has been fixed.. Please review again

File: zsales_class_git.clas.abap (hunk 2 above)
Comment (GIT-REPOSITORY-TEST) [reply zsales_class_git.clas.abap]: The above review comment has been fixed.. Please review again

**Review by Sakthy-g — APPROVED** [review]
Approved for merging with master branch

**Pull request merged**